src/cbc_sdk/audit_remediation/base.py

```diff
@@ -1408,6 +1409,30 @@ def export_zipped_csv(self, filename):
         with io.open(filename, 'wb') as file:
             self.export_csv_as_stream(file, True)
 
+    def async_export(self):
+        """
+        Create an asynchronous job that exports the results from the run.
```

Comment:
We should add a note that this is the recommend approach when expecting a large result set

Reply:
I'm adding this as a documentation change.